### PR TITLE
Groom 1.6.0 release changelog

### PR DESCRIPTION
**Pull request by JamesBurnside**
Groom changelog for 1.6.0 release

### CODE REVIEW

File: packages/communication-react/CHANGELOG.stable.md [hunk 1]
```diff
@@ -6,271 +6,106 @@ This log was last generated on Mon, 12 Jun 2023 19:04:18 GMT and should not be m
...
+- Fixed long `seen` display name as ellipsis ([PR #3090](https://github.com/azure/communication-ui-library/pull/3090) by 77021369+jimchou-dev@users.noreply.github.com)
+- Fix side pane overlapping with composite when window is narrow ([PR #2864](https://github.com/azure/communication-ui-library/pull/2864) by 79475487+mgamis-msft@users.noreply.github.com)
+- Remove side pane control button container if no controls are present to be used ([PR #2969](https://github.com/azure/communication-ui-library/pull/2969) by 94866715+dmceachernmsft@users.noreply.github.com)
+- Add `Edited` for live message speak out ([PR #3173](https://github.com/azure/communication-ui-library/pull/3173) by longamy@microsoft.com)
+- Thematically change hold icon color ([PR #2706](https://github.com/azure/communication-ui-library/pull/2706) by edwardlee@microsoft.com)
+- Fixes error bar messaging for when camera is in use. ([PR #2671](https://github.com/azure/communication-ui-library/pull/2671) by 94866715+dmceachernmsft@users.noreply.github.com)
```
Comment (mgamis-msft) [inline packages/communication-react/CHANGELOG.stable.md:48]: nit: remove period

File: packages/communication-react/CHANGELOG.stable.md [hunk 2]
```diff
@@ -6,271 +6,106 @@ This log was last generated on Mon, 12 Jun 2023 19:04:18 GMT and should not be m
...
+- Add error string for when your remote video feed is frozen for others in the call. ([PR #2808](https://github.com/azure/communication-ui-library/pull/2808) by 94866715+dmceachernmsft@users.noreply.github.com)
+- Fix findDOMNode warning caused by react.strictmode. ([PR #2621](https://github.com/azure/communication-ui-library/pull/2621) by 94866715+dmceachernmsft@users.noreply.github.com)
```
Comment (mgamis-msft) [inline packages/communication-react/CHANGELOG.stable.md:50]: nit: remove period from these 3 change logs and "React.StrictMode" is more proper

File: packages/communication-react/CHANGELOG.stable.md [hunk 3]
```diff
@@ -6,271 +6,106 @@ This log was last generated on Mon, 12 Jun 2023 19:04:18 GMT and should not be m
...
-  - Fix VideoTile to show menu button on hover when isSpeaking prop is true ([PR #2721](https://github.com/azure/communication-ui-library/pull/2721) by 79475487+mgamis-msft@users.noreply.github.com)
-  - Fix scroll and new message prompt in sidepane chat for safari ([PR #3116](https://github.com/azure/communication-ui-library/pull/3116) by edwardlee@microsoft.com)
-  - Update locale strings ([PR #3181](https://github.com/azure/communication-ui-library/pull/3181) by 2684369+JamesBurnside@users.noreply.github.com)
-  - Fixed storybook order and typo ([PR #2940](https://github.com/azure/communication-ui-library/pull/2940) by 109105353+jpeng-ms@users.noreply.github.com)
-  - Bound CallReadiness screens to the boundaries of the app ([PR #2624](https://github.com/azure/communication-ui-library/pull/2624) by edwardlee@microsoft.com)
-  - Default roleHint for AzureCommuncationCallAdapter is 'Consumer' if the locator is for a rooms call ([PR #2942](https://github.com/azure/communication-ui-library/pull/2942) by 79475487+mgamis-msft@users.noreply.github.com)
```
Comment (mgamis-msft) [inline packages/communication-react/CHANGELOG.stable.md:136]: Why was this bug fix changelog removed?

File: packages/communication-react/CHANGELOG.stable.md [hunk 4]
```diff
@@ -6,271 +6,106 @@ This log was last generated on Mon, 12 Jun 2023 19:04:18 GMT and should not be m
...
-  - Video effect errors test ([PR #3066](https://github.com/azure/communication-ui-library/pull/3066) by 97124699+prabhjot-msft@users.noreply.github.com)
+- Switch video seamlessly from different devices ([PR #2726](https://github.com/azure/communication-ui-library/pull/2726) by jinan@microsoft.com)
+- Center more button in video tile ([PR #2902](https://github.com/azure/communication-ui-library/pull/2902) by edwardlee@microsoft.com)
+- Fix messages thread loading issue when an adapter is updated ([PR #2784](https://github.com/azure/communication-ui-library/pull/2784) by 98852890+vhuseinova-msft@users.noreply.github.com)
+- Addressed the issue where local participant label's luminosity ratio does not meet MAS guidelines ([PR #2795](https://github.com/azure/communication-ui-library/pull/2795) by 109105353+jpeng-ms@users.noreply.github.com)
+- Allow horizontal gallery tiles to resize to allow for better use of space. ([PR #2830](https://github.com/azure/communication-ui-library/pull/2830) by 94866715+dmceachernmsft@users.noreply.github.com)
```
Comment (mgamis-msft) [inline packages/communication-react/CHANGELOG.stable.md:35]: nit. Maybe a search of ". (PR" and replace to " (PR" can be done to be consistent and not have periods

File: packages/communication-react/CHANGELOG.stable.md [hunk 5]
```diff
@@ -6,271 +6,106 @@ This log was last generated on Mon, 12 Jun 2023 19:04:18 GMT and should not be m
...
+- Embed the EditBox inside a Chat.Message to enable the accessibility ([PR #3097](https://github.com/azure/communication-ui-library/pull/3097) by longamy@microsoft.com)
+- Update useAzureCommunicationCallAdapter hook to protect against multiple client creations ([PR #3157](https://github.com/azure/communication-ui-library/pull/3157) by 94866715+dmceachernmsft@users.noreply.github.com)
+- Replace Coffee Icon with Spinner ([PR #2885](https://github.com/azure/communication-ui-library/pull/2885) by 97124699+prabhjot-msft@users.noreply.github.com)
+- Update Aria labels for people button to allow for voice access controls on windows. ([PR #2833](https://github.com/azure/communication-ui-library/pull/2833) by 94866715+dmceachernmsft@users.noreply.github.com)
+- Remove default persona coin size in VideoTile. Changed onRenderAvatar callback not show persona until coin size is defined. ([PR #3051](https://github.com/azure/communication-ui-library/pull/3051) by 79475487+mgamis-msft@users.noreply.github.com)
```
Comment (mgamis-msft) [inline packages/communication-react/CHANGELOG.stable.md:59]: Sorry for my bad changelog here. Maybe change to
Changed VideoTile not to show persona coin until an appropriate size is calculated

Comment (JamesBurnside) [reply packages/communication-react/CHANGELOG.stable.md]: Updated

File: packages/communication-react/CHANGELOG.stable.md (hunk 4 above)
Comment (JamesBurnside) [reply packages/communication-react/CHANGELOG.stable.md]: Updated.

File: packages/communication-react/CHANGELOG.stable.md (hunk 2 above)
Comment (JamesBurnside) [reply packages/communication-react/CHANGELOG.stable.md]: Updated.

**Review by mgamis-msft — APPROVED** [review]

**Pull request merged**